terraform/main.tf

```diff
@@ -0,0 +1,194 @@
+provider "aws" {
```

Comment:
If this was an application that would considerably evolve its terraform code over time, how would you structure it?

Reply:
all things should be variables.
if it can be a separate module - create it.